Article/Article.js

```diff
@@ -3,24 +3,24 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = element.querySelector('.expandButton');
```

Comment:
what would be the difference between `element.querySelector('.expandButton')` and `this.element.querySelector('.expandButton')` in this instance?